services/dy-jupyter-extensions/neuron/Dockerfile

```diff
@@ -0,0 +1,84 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} AS common
+LABEL maintainer="guidon"
+
+USER root
+
+# NEURON requirements
+RUN apt-get update &&\
+  apt-get install --no-install-recommends -y \
+    automake \
+    bison \
+    bzip2 \
+    ca-certificates \
+    curl \
+    flex \
+    g++ \
```

Comment:
Do you uninstall some of this stuff afterwards ... must make quite a bit image!

Reply:
Uninstall would not fix the problem I guess because the layer information will still be around, no? Probably a proper multi-stage build would be required.